
This works arround annoying audio glitches due to 200ms in-kernel sleep.

I tried various techniques, including BenH's latest radeonfb rework, which
does even cause new trouble. So I decided to commit just a tiny workarround
for the transition period.

  - Rene Rebe <rene@exactcode.de>

--- ./drivers/video/aty/radeon_base.c~	2004-10-18 23:53:46.000000000 +0200
+++ ./drivers/video/aty/radeon_base.c	2004-11-01 13:47:23.045946776 +0100
@@ -1812,7 +1812,9 @@
 			lvds_gen_cntl &= ~LVDS_BLON;
 			OUTREG(LVDS_GEN_CNTL, lvds_gen_cntl);
 			(void)INREG(LVDS_GEN_CNTL);
-			mdelay(rinfo->panel_info.pwr_delay);/* OUCH !!! FIXME */
+
+			udelay(10); /* Hack, too. -ReneR */
+
 			lvds_gen_cntl |= LVDS_BLON;
 			OUTREG(LVDS_GEN_CNTL, lvds_gen_cntl);
 		}
@@ -1832,7 +1834,9 @@
 				  LVDS_BL_MOD_LEVEL_SHIFT);
 		lvds_gen_cntl |= LVDS_DISPLAY_DIS | LVDS_BLON;
 		OUTREG(LVDS_GEN_CNTL, lvds_gen_cntl);
-		mdelay(rinfo->panel_info.pwr_delay);/* OUCH !!! FIXME */
+
+		udelay(10); /* Hack, too. -ReneR */
+
 		lvds_gen_cntl &= ~(LVDS_ON | LVDS_EN | LVDS_BLON | LVDS_DIGON);
 	}
 
